infra/environments/modl/terraform.tfvars

```diff
@@ -15,3 +15,6 @@ hosted_zone = "curibio-modl.com"
 
 # squarespace dns
 sqsp_verification = "862laeb5r7tfnxngc2mc"
+
+#database
+instance_type = "db.t3.small"
```

Comment:
Probably need to change this eventually (?), but can do that whenever

Reply:
yeah, it's good for now I think